prime-router/docs/how-to-onboard-a-receiver.md

```diff
@@ -0,0 +1,259 @@
+
+The new organization must have at least one `receiver` defined, but the `receiver` does not need to have a transport defined - the download site can be used until an automated delivery mechanism is set up.
+
+There are two fields that will be used as "keys" to further work, below.   The `-name` of the organization (eg, `lt-phd`) will be used as the Okta `group`, and the `translation.schemaName:` value (eg, `lt/lt-covid-19`) will be used as the schema name in the next step.
+
+Below is an example of the organization file
+
+```yaml
+- name: lt-pdh
+  description: LT Department of Health
+  jurisdiction: STATE
+  stateCode: LT
+  receivers:
+    - name: elr
+      topic: covid-19
+      jurisdictionalFilter: [ "matches(ordering_facility_state, LT)" ]
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        receivingApplicationName: LT-PDH
+        receivingApplicationOID: 
+        receivingFacilityName: LT-PDH
+        receivingFacilityOID: 
+```
+
+In the above example, the jurisdictional filter searches the `ordering_facility_state` field in the report for anything that matches the code LT.
+
+In addition, there is the translation section, which specifies the output format that will be sent to the receiver. Currently, we have three formats available:
+
+- HL7 - Each record is converted to HL7 before being sent to the receiver
+- CSV - The records are converted to CSV before being sent to the receiver
+
+The mechanism for how each record is translated is laid out in the schema, which is discussed below
+
+### Set up a new schema
+
+- Canonical location style:   `metadata/schemas/LT/lt-covid-19.schema`
+- Canonical name style: `name: lt-covid-19`
+- Typically, you can copy a basic/standard schema based on a previous state, and modify.
+
+Let's look at a schema header, and then a schema element so you can see an example of each.
+
+#### Schema Header
+```yaml
+---
+name: lt-covid-19
+description: LT COVID-19 HL7
+topic: covid-19
+trackingElement: message_id
+basedOn: covid-19
+elements:
+  # a list of elements follows here
+```
+
+In the example above, the name is what the schema is known by in the system. If you execute the following command in the `prime-router` folder you will see a list of the schemas, clients, and receivers that we currently have loaded in PRIME.
+
+```shell
+./prime list
+```
+`description` is used to give a more descriptive name to the schema.
+
+`topic` ties the schema to a specific topic for routing information. If you refer back to the organization example above, you will see that a topic is provided. These must match. If the topic of the schema does not match the schema being used to translate information for a receiver, you will get an error.
+
+`trackingElement` is the name of the unique key for each row being processed
+
+`basedOn` is one of two ways we can inherit from another schema. The other potential option is `extends`. The difference between the two is subtle but important.
```

Comment:
"subtle but important".  Let me know if you figure out this distinction - I haven't.  ;)